util/src/main/java/com/pkb/pkbcommon/ageutil/AgeUtilService.java

```diff
@@ -0,0 +1,25 @@
+package com.pkb.pkbcommon.ageutil;
+
+import com.pkb.common.datetime.DateTimeService;
+
+import java.time.LocalDate;
+import java.time.ZoneOffset;
+import java.time.temporal.ChronoUnit;
+
+public class AgeUtilService {
+
+    private final DateTimeService dateTimeService;
+
+    public AgeUtilService(DateTimeService dateTimeService) {
+        this.dateTimeService = dateTimeService;
+    }
+
+    public final static int MINIMUM_AGE_LETTER_INVITATION = 16;
```

Comment:
NB: This is likely to be different for different counties etc. Given we're doing the nice thing of pulling it into a shared service, we could also pull the age into PkbConfig?

Reply:
I've moved this. It's now in the config for the FHIR and PHR services